### DISCUSSION TITLE
Actividad Docker - GRUPO 11

### DISCUSSION THREAD

**Discussion by franlucich** [frlp-utn-ingsoft/catedra#116] [Show and tell]
https://github.com/pchesini/vetsoft

**Comment by rodriguezemautn** [comment]
Correcciones:
- el archivo env-example no lleva . (punto)
- de donde obtengo los valores de las variables .env
- los endpoint dan error, no se migra la base de datos(verificar orden del CMD en el Dockerfile o utilizar un script.sh en un ENTRYPOINT) 

Puntos aprobados:
* construye la imagen
* buen tamaño de imagen
* accede a la app (pero los endpoints dan error)
* continua ejecuncion en segundo plano (-d)

Salida de comandos:
$ docker images -a
vetsoft-app   1.0       ff4a6ebb95c7   38 seconds ago   307MB
$ docker ps  -a
3960b8b4aa6f   vetsoft-app:1.0   "python manage.py ru…"   9 seconds ago   Up 9 seconds   0.0.0.0:8000->8000/tcp, :::8000->8000/tcp   Vetsoft

**Comment by fdasilva5** [comment]
Hola, ya hicimos las correcciones que mencionaste.
